app/services/imo_api_service.rb

```diff
@@ -30,9 +30,10 @@ def send_invitation(patient)
     response = execute_post(url, body: request_body)
     result = process_response(response, url, "invitation")
 
-    return if result.nil?
+    return if result == :error
 
-    ImoAuthorization.create!(patient: patient, status: result, last_invited_at: Time.current)
+    status = result == :success ? :invited : result
```

Comment:
Fun - does the invitation API send us a `success` response, that we're translating to something more semantic here?

Reply:
Out of curiosity, what else can `result` be? Anything other than `success` and `error`?

Reply:
It can also be `not_subscribed` which could happen if the patient subscribed then unsubscribed or `no_imo_account` which could happen if the patient deleted their imo.

Reply:
There may be others in the future as we get to know this API better.